analytic_base_department/README.rst

```diff
@@ -16,8 +16,8 @@ No required dependency on Accounting modules.
 
```

Comment:
update whole readme according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst